src/makeRobotAccountant.js

```diff
@@ -6,7 +6,25 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  function getSum(x) {
+    return function(y) {
+      count++;
+
+      if (count > 3) {
+        if (count % 2 !== 0) {
+          return x + y;
+        } else {
+          return `Bzzz... Error!`;
+        }
+      } else {
+        return x + y;
+      }
```

Comment:
using nested conditions is a bad idea, you definitely could have made it more simpler

Reply:
as you can see, you duplicete `return x + y;` – you know what to do 😉